docs/using_eessi/setting_up_environment.md

```diff
@@ -1,6 +1,48 @@
 # Setting up your environment
 
-To set up the EESSI environment, simply run the command:
+To set up the EESSI environment, you can either use EESSI with Lmod or use a bash initialisation script.
```

Comment:
This sounds a bit strange to me "set up the EESSI environment ... by ... _use_ EESSI with Lmod or _use_ a bash ..."

Maybe use different verbs and objects? For example
```suggestion
To set up the EESSI environment, you can either load an EESSI module with Lmod or source a bash initialisation script.
```